### ISSUE TITLE
UI/API: make display elements optional

### ISSUE THREAD

**Issue by DaanHoogland** [apache/cloudstack#6855]
##### ISSUE TYPE

 * Improvement Request

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.17
~~~

##### SUMMARY

Whenever there is a `name` and a `displayName` or a `name` and a `displayText` or `name` and `description` and both are required, please make the `display` field optional and let the default be the contents of the `name`

- [x] create network
- [ ]    create disk offering
- [ ]   create service (system, compute) offering
- [ ]    create network offering
- [ ]    create vpcoffering
- [ ]    create project
- [ ]    create vpc
- [ ]    create template from volume
- [ ]    register/upload template
- [ ]    register/upload ISO

##### STEPS TO REPRODUCE

~~~
start the new network wizard
~~~

##### EXPECTED RESULTS

~~~
i can type `newNet` in the name field and it will result in a net with `name` and `description` both being that value
~~~

##### ACTUAL RESULTS

~~~
both fields are required and I have to copy paste the value from one to the other.
~~~

**Comment by sanjeev98kumar** [comment]
I am looking for an issue for academic contribution. Will this be a good first issue?

**Comment by DaanHoogland** [comment]
It is not a very difficult one, but does require some knowledge and understanding of both the system and the UI.

**Comment by rohityadavcloud** [comment]
with @rahulbcn27 attempting it cc @DaanHoogland @weizhouapache

**Comment by rahulbcn27** [comment]
Thanks @rohityadavcloud .Working on it

**Comment by rahulbcn27** [comment]
Thanks @DaanHoogland for updating the list.

**Comment by weizhouapache** [comment]
fixed via #7112 and #7180
